### ISSUE TITLE
Button Groups needs updated to match SLDS Examples

### ISSUE THREAD

**Issue by arodenbeck** [salesforce/design-system-react#1229]
The 3rd button in the 2nd example should be changed to match SLDS in its appearance, and needs button text.
